test/evpdigest.txt

```diff
@@ -90,7 +91,8 @@ Digest = BLAKE2b512
 Input = 000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F202122232425262728292A2B2C2D2E2F303132333435363738393A3B3C3D3E3F404142434445464748494A4B4C4D4E4F505152535455565758595A5B5C5D5E5F606162636465666768696A6B6C6D6E6F707172737475767778797A7B7C7D7E7F8081
 Output = DF0A9D0C212843A6A934E3902B2DD30D17FBA5F969D2030B12A546D8A6A45E80CF5635F071F0452E9C919275DA99BED51EB1173C1AF0518726B75B0EC3BAE2B5
 
-# SHA tests from RFC6234 section 8.5
+Title = SHA(1) tests (from shatest.c)
```

Comment:
shatest.c is not present anymore, no point to mention it. I'd suggest just "SHA1 tests" here and either "SHA2 tests" prior SHA224/256/384/512 or separate SHAxxx titles...